### PR TITLE
Xsup 39719 - run script mid results

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-39719

## Description
Added the action id and other execution metadata to the xdr-script-run command partial result to be available to customers in case the polling times out.

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
Failing TPBs are known here:
- https://jira-dc.paloaltonetworks.com/browse/CIAC-11274
- https://jira-dc.paloaltonetworks.com/browse/CIAC-11223

Validation error is by design - we don't want a special release for core pack.

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py [hunk 1]
```diff
@@ -3319,6 +3319,10 @@ def script_run_polling_command(args: dict, client: CoreClient) -> PollResult:
             response=None,  # since polling defaults to true, no need to deliver response here
             continue_to_poll=True,  # if an error is raised from the api, an exception will be raised
             partial_result=CommandResults(
+                outputs_prefix=f'{args.get("integration_context_brand", "CoreApiModule")}.ScriptRun',
+                outputs_key_field='action_id',
+                outputs=reply,
```
Comment (JudahSchwartz) [inline Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:3324]: will we be returning the same information several times now?

Comment (ShahafBenYakir) [reply Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:3324]: Only in the first run. In the rest of the runs we'll have action_id so another flow will run 
https://github.com/demisto/content/pull/35575/files#diff-d973b6667309bcba22ae2bbc58be0bb9b6a44750b343545c4b1654a23c3029deR3299

**Review by JudahSchwartz — APPROVED** [review]
nice!

**Pull request merged**